libr/core/cmd_anal.c

```diff
@@ -1182,12 +1205,25 @@ static int var_cmd(RCore *core, const char *str) {
 	RAnalFunction *fcn = r_anal_get_fcn_in (core->anal, core->offset, -1);
 	/* Variable access CFvs = set fun var */
 	switch (str[0]) {
-	case '-':
-		// "afv"
+	case '-': // "afv-"
 		r_core_cmdf (core, "afvs-%s", str + 1);
 		r_core_cmdf (core, "afvb-%s", str + 1);
 		r_core_cmdf (core, "afvr-%s", str + 1);
 		return true;
+	case 'x': // "afvx"
+		{
+		RList *list = r_anal_get_functions_in (core->anal, core->offset);
```

Comment:
exact this in a separate function `afvx_handler (core)` and call it to implement `axv` instead of calling `r_core_cmd`

Reply:
about afv= i think it should die. the subcommand makes no sense, and afvx is more meaningful because it refers to 'x'refs. and = is used for lsiting things in bars format. so im gonna kill it now